test/e2e_node/runner/e2e/run_e2e.go

```diff
@@ -34,7 +34,7 @@ import (
 	"sync"
 	"time"
 
```

Comment:
That's fine, except I recommended moving `pkg/remote` to `runner/remote` (but maybe they could be in the same package?).